lib/DDG/Spice/Crackhash.pm

```diff
@@ -0,0 +1,20 @@
+package DDG::Spice::Crackhash;
+
+use DDG::Spice;
+use strict; 
+
+spice is_cached => 0;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 0; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+spice to => 'http://api.md5crack.com/crack/ZFCQCcpS1kLgbCQZ/$1';
+
+triggers any => 'crackmyhash', 'crackhash' , 'crack hash' , 'crack';
+
+handle remainder => sub {
+   
+    # Query is in $_...if you need to do something with it before returning
```

Comment:
To add to that, I think `crack` is a bit too generic, at the very least we want there to be `hash` or `md5` or similar in the query before we accept it.